libs/shared/ui/nuverial/src/lib/components/unsaved-step-modal/unsaved-step-modal.component.scss

```
@@ -6,6 +6,10 @@
     padding: 40px;
     max-width: 400px;
 
+    @media screen and (max-width: 600px) {
```

We have a variable for doing this
```suggestion
@media screen and (max-width: themeVars.$layout-mobile-breakpoint) {
```